packages/connect/src/core/index.ts

```diff
@@ -317,7 +317,7 @@ const inner = async (method: AbstractMethod<any>, device: Device) => {
         }
     }
 
-    const deviceNeedsBackup = device.features.needs_backup;
+    const deviceNeedsBackup = device.features.backup_availability === 'Required';
```

Comment:
note for me: is it backwards compatible?

Reply:
Shall be, `needs_backup` was renamed to this 
![image](https://github.com/trezor/trezor-suite/assets/152899911/2cc24c05-9252-4671-a485-a57cc9a7b9ad)

and the enum is done in back compatible way
![image](https://github.com/trezor/trezor-suite/assets/152899911/44c58409-a09b-4e58-a8ce-27db47f52ebb)

`Required = 1` now replaces the `true` in the original bool type,

Reply:
and what happend with the `needs_backup` field?

Reply:
![image](https://github.com/trezor/trezor-suite/assets/152899911/4b74ea03-39ed-427a-b0ee-9f988e5a5a3f)